### PR TITLE
feat(web): link to combined approved-SP dashboard on landing

### LINKED ISSUE #384: Dealbot landing page links to a view of combined approved SP metrics
## Done Criteria
We expose a way on the dealbot landing page (e.g., dealbot.filoz.org) to view the combined metrics of all approved SPs (i.e., betterstack sp dashboard for approved SPs)

## Why Important
We want to push towards clients interacting with and thinking about FOC as a product rather than building individual relationships with specific SPs (even though that is happening under the covers). Linking to "here is the approved SP performance dashboard" gives a sense of what someone will get when interfacing with FOC on the golden path of using the default approved SPs.
 
## User/Customer
Users trying to get a sense of the overall experience/performance of using FOC to store data.

## Notes

### PR DESCRIPTION
**Pull request by SgtPooki**
## What changed

Adds a link on the landing page to the BetterStack dashboard showing combined performance metrics for approved SPs. Helps visitors see overall FOC storage experience without first picking an individual SP.

New config: `APPROVED_SP_DASHBOARD_URL` (runtime env, consumed via `window.__DEALBOT_CONFIG__`) and `VITE_APPROVED_SP_DASHBOARD_URL` (build-time fallback). Link is hidden when unset; invalid URLs show a warning. Wired into `docker-entrypoint.sh` and the local kustomize overlay (mainnet value).

Closes #384

## How to verify

1. `cd apps/web && pnpm dev`
2. Set `VITE_APPROVED_SP_DASHBOARD_URL` in `.env` (calibration: `https://telemetry.betterstack.com/dashboards/wt6QCw?rf=now-3d&rt=now`; mainnet: `https://telemetry.betterstack.com/dashboards/Zz8k7L?rf=now-72h&rt=now`).
3. Visit http://localhost:5173/ — "View combined approved-SP performance dashboard" link appears in hero.
4. Unset the var and reload — link disappears, no warning.

Validated locally against `VITE_API_BASE_URL=https://staging.dealbot.filoz.org` on calibration.

## Notes

Staging/prod deployments need `APPROVED_SP_DASHBOARD_URL` set per network (calibration → `wt6QCw`, mainnet → `Zz8k7L`).

## Screenshot:

image

### CODE REVIEW

File: apps/web/src/pages/Landing.tsx [hunk 1]
```diff
@@ -103,6 +110,24 @@ export default function Landing() {
...
+              className="inline-flex items-center gap-1 text-primary underline-offset-4 hover:underline font-medium"
+            >
+              View combined approved-SP performance dashboard
+              <ExternalLink className="h-3 w-3" />
+            </a>
+          </p>
+        )}
+        {approvedSpDashboardUrlInvalid && (
+          <p className="text-sm text-yellow-600">
+            Warning: APPROVED_SP_DASHBOARD_URL configured but invalid — link unavailable.
```
[the remark this answers is not in the document]
Comment (SgtPooki) [reply apps/web/src/pages/Landing.tsx]: Addressed in 2c95f7f and follow-up: warning now names both `APPROVED_SP_DASHBOARD_URL_{NETWORK}` and `VITE_APPROVED_SP_DASHBOARD_URL_{NETWORK}` so operators can find the right knob during local dev vs container deploy.

File: apps/web/src/pages/Landing.tsx [hunk 2]
```diff
@@ -103,6 +110,24 @@ export default function Landing() {
...
+          </p>
+        )}
```
[the remark this answers is not in the document]
Comment (SgtPooki) [reply apps/web/src/pages/Landing.tsx]: Intentionally skipping tests for these branches. The render-when-set, hide-when-unset, warning-on-invalid logic is a thin wrapper around the existing `getConfigUrl` helper which already governs DASHBOARD_URL/LOGS_URL without dedicated tests. Adding three more snapshot-ish tests for the same pattern duplicates coverage without protecting against likely regressions. If the URL handling grows non-trivial (filters, transforms), I'll add tests at that point.

**Review by juliangruber — APPROVED** [review]
